src/system/SystemConfig.h

```diff
@@ -765,7 +765,7 @@ struct LwIPEvent;
  *  Defaults to enabled on Zephyr platforms that enable CONFIG_EVENTFD.
  */
 #ifndef CHIP_SYSTEM_CONFIG_USE_ZEPHYR_EVENTFD
-#if defined(__ZEPHYR__) && CONFIG_EVENTFD
+#if defined(__ZEPHYR__) && defined(CONFIG_EVENTFD)
```

Comment:
CONFIG_EVENTFT is a Kconfig setting, right?  See questions about Kconfig contract above.

Reply:
Boolean Kconfig macros are either defined as true or not defined at all.